### PR TITLE
GH-1535: Robustify WordEmbeddingsStore

### PR DESCRIPTION
**Pull request by oterrier**
#1535

### CODE REVIEW

File: flair/inference_utils.py [hunk 1]
```diff
@@ -128,64 +133,70 @@ def delete_stores(model, backend='sqlite'):
...
 
 class WordEmbeddingsStoreBackend:
     def __init__(self, embedding, backend, verbose=True):
         # get db filename from embedding name
         self.name = embedding.name
-        self.store_filename = WordEmbeddingsStore._get_store_filename(embedding, backend)
+        self.store_path : Path = WordEmbeddingsStore._get_store_path(embedding, backend)
         if verbose:
-            print("store filename:", self.store_filename)
+            log.info("store filename:", str(self.store_path))
```
Comment (alanakbik) [inline flair/inference_utils.py:150]: This is causing logging errors on my machine. Can you replace this with 
```python
log.info("store filename: " + str(self.store_path))
```

**Comment by alanakbik** [comment]
@oterrier thanks for adding this! Can you fix the logging issue, then I'll merge!

**Comment by oterrier** [comment]
Sure, I'm wondering how is it possible that I didn't see that during my
tests
I have an additional modification to enable the same mechanism for
TextClassifier

I will do that on monday

Have a nice we

Olivier
Kairntech

Le ven. 24 avr. 2020 à 18:10, Alan Akbik <notifications@github.com> a
écrit :

**Comment by alanakbik** [comment]
@oterrier thanks for adding this!

**Pull request merged**